### PR TITLE
User-authentication

### PR DESCRIPTION
**Pull request by muhammadqasim21**
Created Responsive user Signup and Login. Implemented backend in express and used mysql for the database

Screenshot (312)
Screenshot (313)

### CODE REVIEW

**Review by anssidhu — APPROVED** [review]

File: frontend/src/Components/Login/Input.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Input = ({ id, type, label,required, disabled, onChange }) => (
+    <input className="form-group__input" type={type} id={id} placeholder={label} required={required} disabled={disabled} onChange={onChange} />
+);
```
Comment (Urwa-Ihtesham) [inline frontend/src/Components/Login/Input.js:5]: Is it working with backend?

**Review by Urwa-Ihtesham — APPROVED** [review]

**Review by UrwaIhtesham — APPROVED** [review]

**Review by azubair-experlabs — APPROVED** [review]
Try to add screenshots in PR description whenever you make UI changes

**Review by azubair-experlabs — COMMENTED** [review]
One more thing, the commit messages says "user-authentication" only. It could be more self explanatory like "Authentication : Implemented user login using JWT/credentials/oAuth"

**Comment by muhammadqasim21** [comment]
> Try to add screenshots in PR description whenever you make UI changes

Noted.

**Comment by muhammadqasim21** [comment]
> One more thing, the commit messages says "user-authentication" only. It could be more self explanatory like "Authentication : Implemented user login using JWT/credentials/oAuth"

Okay !

**Comment by SabaSaleem5317** [comment]
we shud use diff Background image and color for login signup.

**Review by SabaSaleem5317 — APPROVED** [review]

**Comment by NoorFatyma** [comment]
Background color should be white and the text color should me maroon so the redness on the page will reduce.

**Review by NoorFatyma — APPROVED** [review]

**Pull request merged**